src/lib/components/identity/identity-manager.svelte

```diff
@@ -171,6 +201,18 @@
         loading = false
     }
 
+    /**
+     * Toggle if identities should be filtered by current user
+     */
+    function onOnlyOwnIdentities(): void {
+        const currentFilter = get(identityFilterOptions)
+        const creatorFilterState = !currentFilter.creatorFilterState
+        // Set local filter value depending on state in store
+        identityFilter[0].value = creatorFilterState ? get(authenticatedUserDID) : undefined
```

Comment:
this can be removed see streamsFilter

Reply:
Still necessary, same reason as in streams manager.